context/php/conf.d/opcache.ini

```diff
@@ -0,0 +1,13 @@
+;zend_extension=opcache.so
+opcache.enable=1
```

Comment:
Please clarify with @lazystar if we can leave `opcache.enable` set to 1, or do we need to take care about disabling opcache while using xdebug. There were issues reported by internal/external customers, when using both xdebug and opcache at the same time.

Reply:
(this can be possibly implemented later)

Reply:
We have different op cache config for dev mode. Anyway, this cannot be solved in this repository.